jfrog-cli/artifactory/commands/config.go

```diff
@@ -145,20 +137,44 @@ func getConfigurationFromUser(details, defaultDetails *config.ArtifactoryDetails
 	return nil
 }
 
-func readSshKeyPathFromConsole(details, savedDetails *config.ArtifactoryDetails) error {
+func getSshKeyPath(details *config.ArtifactoryDetails) error {
+	// If path not provided as a key, read from console:
 	if details.SshKeyPath == "" {
-		ioutils.ScanFromConsole("SSH key file path", &details.SshKeyPath, savedDetails.SshKeyPath)
+		ioutils.ScanFromConsole("SSH key file path (optional)", &details.SshKeyPath, "")
+	}
+
+	// If path still not provided, return and warn about relying on agent.
+	if details.SshKeyPath == "" {
+		log.Info("SSH Key path not provided. Specify a key path using flags, or rely on ssh-agent only.")
```

Comment:
I suggest rephrasing the message to:
SSH Key path not provided. You can also specify a key path using the --ssh-ket-path command option.